solr/webapp/web/js/angular/services.js

```diff
@@ -58,10 +58,10 @@ solrAdminServices.factory('System',
   }])
 .factory('Logging',
   ['$resource', function($resource) {
-    return $resource('admin/info/logging', {'wt':'json', 'nodes': 'all', '_':Date.now()}, {
+    return $resource('admin/info/logging', {'wt':'json', '_':Date.now()}, {
       "events": {params: {since:'0'}},
       "levels": {},
-      "setLevel": {}
+      "setLevel": {'nodes': 'all'}
```

Comment:
The JavaScript framework we use here is completely unknown to me.  Does this change make sense to you @janhoy ?

Reply:
It’s Angular 1. Looks right but I’m not sure if there should be a `params: ` there as well. Testing will tell..

Reply:
I debugged it on Admin UI and it works without `params`. But I updated my PR to use `params` as other handlers also use `params` in this file .